wrapper/src/main/java/software/amazon/jdbc/DataSourceConnectionProvider.java

```diff
@@ -29,6 +29,7 @@
 import org.checkerframework.checker.nullness.qual.NonNull;
 import software.amazon.jdbc.dialect.Dialect;
 import software.amazon.jdbc.exceptions.SQLLoginException;
+import software.amazon.jdbc.plugin.readwritesplitting.ReadWriteSplittingPlugin;
```

Comment:
This class should not rely on particular plugins since it may be used with no such plugin. Need to review why such import is required.